sites/public/src/components/account/SignUpBenefits.tsx

```diff
@@ -0,0 +1,33 @@
+import { Icon } from "@bloom-housing/ui-seeds"
+import { faStopwatch, faEye, faLock } from "@fortawesome/free-solid-svg-icons"
+import { t } from "@bloom-housing/ui-components"
+
+type SignUpBenefitsProps = {
+  className?: string
+  idTag?: string
+}
+const SignUpBenefits = (props: SignUpBenefitsProps) => {
+  const iconListItems = [
+    { icon: faStopwatch, text: t("account.signUpSaveTime.applyFaster") },
+    { icon: faEye, text: t("account.signUpSaveTime.checkStatus") },
+    { icon: faLock, text: t("account.signUpSaveTime.resetPassword") },
+  ]
+  const classNames = ["flex flex-col grow-0 shrink-1 pt-6 pb-6 pr-4 pl-4 md:p-0"]
```

Comment:
Similar callout from Slack convo that this is all good to keep for now since the work is already finished, but moving forward to not use tailwind styles but apply scss in module files.